arborist/server.go

```diff
@@ -52,6 +54,11 @@ func (server *Server) Init() (*Server, error) {
 	if server.logger == nil {
 		return nil, errors.New("arborist server initialized without logger")
 	}
+	stmt, err := server.db.Prepare(AUTH_QUERY)
+	if err != nil {
+		return nil, err
+	}
+	server.authQuery = stmt
```

Comment:
the prepared statements are a good idea, I think let's add one more layer here so it will be more organized if we add many more? just something like this
```go
type preparedStmts struct {
    authQuery *sql.Stmt
}

type Server struct {
    // ...
    stmts preparedStmts
}
```

Reply:
Yeah, sure! I was thinking about similar things, I thought we can do so when needed. But anyway, I'll add this here . ;)